scripts/mutect2_wdl/mutect2.wdl

```diff
@@ -80,48 +80,13 @@ task M2 {
   }
 }
 
-task CalculateContamination {
```

Comment:
Why are you making any changes in mutect2.wdl?    How is it okay to remove the contamination calculation from mutect WDL?

Please explain....